DQM/CTPPS/test/dqm_test_cfg.py

```diff
@@ -50,7 +50,7 @@
 
 # RP geometry
 process.load("Geometry.VeryForwardGeometry.geometryRP_cfi")
-process.XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
+process.XMLIdealGeometryESSource_CTPPS.geomXMLFiles.append("Geometry/VeryForwardData/data/2016_ctpps_15sigma_margin0/RP_Dist_Beam_Cent.xml")
```

Comment:
This config is removed in https://github.com/cms-sw/cmssw/pull/14836/commits/d787b3173d0a908c3afda266a6aaf34fa1b2daba which is part of #14836. There are still problems with a replacement for this config - being discussed with @dmitrijus .